src/frontend/lib/mapbox.ts

```diff
@@ -9,7 +15,7 @@ const config = {
 
 const help = "See https://www.mapbox.com/api-documentation/#access-tokens";
 
-function makeAPIURL(urlObject, accessToken) {
+function makeAPIURL(urlObject: UrlObject, accessToken: string | null = null) {
```

Comment:
what is null = null for?

Reply:
we're setting the default value to `null` here i.e. `...accessToken = null }...`

Reply:
> we're setting the default value to `null` here i.e. `...accessToken = null }...`

Oh I understand = `accessToken : (string|null) = null`